### ISSUE TITLE
Sprites on each part of compound Body

### ISSUE THREAD

**Issue by mturneruk** [liabru/matter-js#926]
Hi there.

Thanks for a fantastic engine.
I have hit a problem in my game development.

I am trying to make a compound spaceship made up of various parts.

var CenterX = game.width/2;
var CenterY = game.height/2;

var WingLeft = Bodies.fromVertices(CenterX-50,CenterY,[{ "x":0, "y":0 }, { "x":0, "y":100 }, { "x":100, "y":100 }, { "x":100, "y":0 }]);
var WingRight = Bodies.fromVertices(CenterX+50,CenterY,[{ "x":0, "y":0 }, { "x":0, "y":100 }, { "x":100, "y":100 }, { "x":100, "y":0 }]});

var Hull....
var Fins....
etc

var Player = Body.create({parts: [WingLeft], WingRight],......});
World.add(engine.world,[Player]);

That was all fine.

Then I started adding sprite textures to the parts.. e.g.

var WingRight = Bodies.fromVertices(CenterX+50,CenterY,[{ "x":0, "y":0 }, { "x":0, "y":100 }, { "x":100, "y":100 }, { "x":100, "y":0 }], {
    render: {
         sprite: {
             texture: 'assets/wingR.png'
         }
    }
});

I can see the textures, but now when my ship crashes into a wall body the ship breaks apart as though the parts are not compound. I have tried making my sprites smaller than the size of the body.

Am I doing something wrong, or does matter not support sprites on parts.
Is there any way around this ?

Also when i want the ship to explode is there a way to make a compound Body break apart ?

Cheers for any help you can give

**Comment by liabru** [comment]
What do you mean by the ship breaks apart? Compound bodies should not destruct on collision with anything, so could you explain further? 

One thing to know is that changing the parts of compound bodies after creation is not supported by default, as there would be a number of steps involved, so I'd suggest creating a new body to replace it if that's what you need to do.

**Comment by mturneruk** [comment]
Hey - Nice to hear from a matter God!... Thanks for great engine.

I have since started using phaser 3 with matter and I have managed to achieve that by using separate matter sprite objects with matter bodies, and moving and rotating each part individually, so the main ship looks like one object. 
When the ship is destroyed, each part of the ship is given a random velocity in a random direction so it appears to break up..

My biggest current issue is that i am struggling to limit the matter update rate on a 144Hz monitor.
Don't suppose you know of a way ?

Thanks again. Are you still developing Matter - Its brilliant?
